src/cli.js

```diff
@@ -90,7 +89,7 @@ function findConfig(file) {
   else if (envs) {
     home = path.normalize(path.join(envs, ".jshintrc"));
 
-    if (shjs.test("-e", home))
+    if (fsUtils.exists(home))
```

Comment:
Why not use  `fs.existsSync()`?

Reply:
This method is backed by `fs.existsSync`. The reason not to use the Node.js method directly is so that we can more easily substitute in an implementation designed for the Rhino runtime (I'd like to restore support for Rhino one day).